modules/upload/client-react/actions/uploadFiles.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { ActionType } from '../reducers';
+import { ActionFunction } from '.';
+
+const UPLOAD_FILES: ActionFunction = (files: any) => ({
```

Comment:
I have never seen functions to be called like this. Could you please give me a link to this convention.

Reply:
Fixed